docs/platforms/javascript/guides/nuxt/install/dynamic-import.mdx

```diff
@@ -28,8 +40,11 @@ import('./chunks/nitro/nitro.mjs').then(function (n) { return n.r; });
 
 ## Scenarios where `import()` doesn't work
 
-We are currently investigating an issue where the server-side is not correctly initialized with a recent update of Nitro (the server-side toolkit in Nuxt).
-We are working on figuring this out ([see issue here](https://github.com/getsentry/sentry-javascript/issues/14514)). As a temporary workaround, you can add the following overrides to your application:
+Depending on your setup and which version of Nitro is used, the server-side is sometimes not correctly initialized.
+The build output **must not** include a regular `import` of the Nitro runtime (e.g. `import './chunks/nitro/nitro.mjs'`).
```

Comment:
m: I think we need a simpler way of telling people how to detect this other than looking through some build files they might not even have access to.

How about monitoring startup logs and detecting whether otel wrapped http instead?

I think the technical details are better kept in the github issue linked below.

Reply:
https://github.com/getsentry/sentry-javascript/issues/14638

I added an issue for this and will update the docs after adding this warning 👍